web/src/components/SubscribeDialog.js

```diff
@@ -118,7 +131,8 @@ const SubscribePage = (props) => {
                         maxLength: 64,
                         "aria-label": t("subscribe_dialog_subscribe_topic_placeholder")
                     }}
-                />
+                    />
+                <Button onClick={generateTopicName} disabled={props.topic.includes("-")}>{t("subscribe_dialog_subscribe_button_generate_topic_name")}</Button><br />
```

Comment:
The button should in the same line as the text field, IMHO.

Reply:
Sure, done